frontend/src/components/sidebar/MenuItemComponent.js

```diff
@@ -53,8 +53,8 @@ function MenuItemComponent({ children, icon: Icon, id, items = [], level = 1, on
     const classNameColumn = isActive ? classes.leftBar : '';
     const classNameContainer = [classes.container, isActive && classes.activeContainer].join(' ');
     const iconColor = isActive
-        ? theme.uniformStyle.color.highlightingColor
-        : theme.uniformStyle.color.primaryIconColor;
+        ? '#00b300'
+        : '#b5b4b9';
```

Comment:
How can we use the styleSheet (theme.uniformStyle.color.highlightingColor) in css? it is an will be a hassle to do it like this with all the color codes in 1000 different places

Reply:
I am not using the color codes we defined previously due to the fact that we can not use them in .css . And regarding your question , I guess we need to figure it out during this sprint. I havent try it yet.